### DISCUSSION TITLE
What's next ? randomize RE1 2 3 rdt files in one game ?

### DISCUSSION THREAD

**Discussion by otamega72** [IntelOrca/biorand#534] [General]
Will biorand future version randomize RE1 2 3 rdt files in one game ?

Imagine playing RE2 , open a door , enter into RE1 mansion or RE3 street .....

**Comment by IntelOrca** [comment]
Theoretically possible with the following limits:
* You are still limited by a certain number of rooms each game, so other game RDTs would replace existing ones after filling the spare slots.
* The key items would be specific to one game, so not sure how that would work with other RDTs. Replacing various key items could get complicated.
* The room script would need to be re-written in a way that is compatible, doesn't rely on flags and stuff from other rooms.

It would be better to wait for a full re-write of RE 2 where these limits can removed and you can have all 3 games in one engine.
